### DISCUSSION TITLE
Published GitHub Page

### DISCUSSION THREAD

**Discussion by JPJimrah** [community/community#27724] [New to GitHub] [answered]
WeMy published github page is not showing its content. Please, what could be the problem?
Repo link:
https://github.com/JPJimrah/Responsive-Registration-Form
Page link:
https://jpjimrah.github.io/Responsive-Registration-Form/

**Comment by airtower-luna** [comment, marked as answer]
1. Your repository doesn't have an `index.html` file, so if you follow the page link you'll see a rendering of the README. If you go to https://jpjimrah.github.io/Responsive-Registration-Form/registration_form.html you get something resembling a form.
2. The `registration_form.html` looks like you committed Git conflict markers. You should resolve the conflict, remove the markers, and commit the content you actually want. The conflict markers are these:
```
<<<<<<< HEAD
<!DOCTYPE html>
...
</body>
=======
<!DOCTYPE html>
...
</body>
>>>>>>> 21a0e242d9a91ab6b5eaa4fe6d3158646ac7b610
</html>
```
3. If you want to see `registration_form.html` when you open the page link, rename it to `index.html`. You might also want to add an empty `.nojekyll` file [to disable the default Jekyll processing that used the README](https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages#static-site-generators).

**Comment by JPJimrah** [comment]
How do I remove the conflict markers in no. 2 above?

**Reply by PythonCoderAS** [reply]
You can also use a git visual client (like VSCode) which has a git merge conflict resolver.

**Reply by EduardoP103** [reply]
You can use git software

**Reply by JPJimrah** [reply]
OK. Thanks

**Comment by airtower-luna** [comment]
They are part of your file, use your editor of choice (and then commit and push), or the editor on GitHub if you prefer. You can easily delete the lines, the real question is which version of the code in between you want to keep (possibly a combination).

**Reply by JPJimrah** [reply]
Oh, Thanks for the tip!
